.travis.yml

```diff
@@ -82,7 +82,7 @@ script:
   - if [[ $NO_DEBUGGER == 1 ]]; then xdebug-disable; fi
   - |
     if [[ $PHPDBG != 1 ]]; then
-      bin/infection $INFECTION_FLAGS;
+      INFECTION_ALLOW_XDEBUG=1 bin/infection $INFECTION_FLAGS;
```

Comment:
could you please explain why do we need to run xdebug? 
if you check `$INFECTION_FLAGS` you will see that we put coverage path there. We did it on purpose to speed up travis builds.

Reply:
See issue #344 and comments starting with this https://github.com/infection/infection/issues/344#issuecomment-388598387

It is not like that's the only way to go. Another way is to ignore issue #344 completely, since we're getting valid metrics from phpdbg runs.